packages/studio-ui/src/components/AddPageButton/AddPageButton.tsx

```diff
@@ -38,7 +38,16 @@ function AddPageButtonInternal() {
           setStep(FlowStep.GetBasicPageData);
           break;
         case FlowStep.GetBasicPageData:
-          await createPage(pageName, getPathVal, state.streamScope);
+          setStep(FlowStep.SelectLayout);
+          break;
+        case FlowStep.SelectLayout:
+          state.pageName &&
+            (await createPage(
+              state.pageName,
```

Comment:
Could we do `state.pageName || ""` here and get rid of the `&&` conditional above? I think that reads a bit cleaner. Really, we know the `state.pageName` is set here. You can't get to this part of the flow without it. Ultimately, we should figure out how we can get rid of the empty string defaulting here. But, that doesn't need to be part of this PR. We were doing that defaulting before.